.github/scripts/update_dev_packages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+set -e
+
+core_git_branch=$1
```

Comment:
Since ultimately we'll remove `dbt-core` as a dependency, can we make `core_git_branch` the second argument and `adapters_git_branch` the first argument?